python_modules/libraries/dagster-tableau/dagster_tableau/resources.py

```diff
@@ -542,10 +542,14 @@ def fetch_tableau_workspace_data(
                             data_source_id = published_data_source_data["luid"]
                             if data_source_id and data_source_id not in data_source_ids:
                                 data_source_ids.add(data_source_id)
+                                augmented_published_data_source_data = {
+                                    **published_data_source_data,
+                                    "isPublished": True,
```

Comment:
Using "isPublished" instead of "is_published" to respect convention of the Tableau data we receive at this stage